package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@langleyfoxall/react-dynamic-data-table",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
```suggestion
  "version": "2.3.0",
```

There is not need to update the version, as this is not a bug fix, new feature, or breaking cahnge.

Reply:
NPM won't show the read me unless you publish, you can't publish the same version twice

Reply:
Fair enough. Let's bump the version as you suggested.

Reply:
It's just annoying that the readme isn't pulled from the repo, such as composer.